lxd/network/driver_common.go

```diff
@@ -578,15 +578,22 @@ func (n *common) handleDependencyChange(netName string, netConfig map[string]str
 func (n *common) bgpValidationRules(config map[string]string) (map[string]func(value string) error, error) {
 	rules := map[string]func(value string) error{}
 	for k := range config {
-		// BGP keys have the peer name in their name, extract the suffix.
+		// only BGP keys are handled here.
 		if !strings.HasPrefix(k, "bgp.") {
 			continue
 		}
 
-		// Validate remote name in key.
+		switch k {
+		case "bgp.ipv4.nexthop":
```

Comment:
We also need to `continue` the loop here if either bgp.ipv4.nexthop or bgp.ipv6.nexthop.